relayer-config-example.toml

```diff
@@ -12,6 +12,11 @@ consumer_description = "Event processor"
 deliver_group = "event-processors"
 
 [gcp]
+amplifier_tls_public_certificate: '''
```

Comment:
```suggestion
amplifier_tls_public_certificate = '''
```

Reply:
Is it different than the `tls_public_certificate` under `[amplifier_component]`?

Reply:
yeah I removed it